{{cookiecutter.project_slug}}/Makefile

```diff
@@ -34,7 +37,8 @@ CUSTOM_ENV=image:neuromation-$(PROJECT):$(VERSION)
 # Allows to set the `neuro` executable:
 #   make setup NEURO=/usr/bin/neuro
 #   make setup NEURO="neuro --verbose --show-traceback"
-NEURO?=neuro
+NEURO=neuro
```

Comment:
Just to clarify: if you set a variable with `=`, doesn't that mean that you won't be able to overwrite it with `make setup NEURO=/usr/bin/neuro`?

Reply:
OK good question. Here's the reference: https://www.gnu.org/software/make/manual/html_node/Setting.html#Setting
- `=`, `:=`, or `::=` define variable always:
```
[ay@archlinux temp]$ cat Makefile 
VAR=default
target:
	@echo $(VAR)

[ay@archlinux temp]$ make target 
default
[ay@archlinux temp]$ make target VAR=value1
value1
[ay@archlinux temp]$ export VAR=value2
[ay@archlinux temp]$ make target
default
```
note the exported variable did not override the value set in Makefile.

- `If you’d like a variable to be set to a value only if it’s not already set, then you can use the shorthand operator ‘?=’ instead of ‘=’.`:
```
[ay@archlinux temp]$ unset VAR
[ay@archlinux temp]$ cat Makefile 
VAR?=default
target:
	@echo $(VAR)

[ay@archlinux temp]$ make target VAR=value3
value3
[ay@archlinux temp]$ make target 
value5
[ay@archlinux temp]$ make target VAR=value6
value6
```

So we mark `?=` variables that act as target arguments (`make train RUN=value`, `make train PRESET=cpu-small`), and we mark `=` variables that are not supposed to be inline-overriden by the user (`make train PROJECT_ID=value` is discouraged).